src/utils/utils.ts

```diff
@@ -0,0 +1,102 @@
+import {BaseElementsDto} from './base-elements.dto';
+import {CourseDto} from './course.dto';
+import {BookDto} from './book.dto';
+import {ConferenceDto} from './conference.dto';
+
+export class Utils {
+ private static BASE: BaseElementsDto[] = [
```

Comment:
for mock data can be export const like
```suggestion
 export const BASE: BaseElementsDto[] = [
```